server2/Arcadia.Assistant/Arcadia.Assistant.NotificationTemplates/DictionaryExtensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Arcadia.Assistant.NotificationTemplates
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using System.Runtime.Serialization;
+
+    public static class DictionaryExtension
+    {
+        public static Dictionary<T1, T2> Perform<T1, T2>(
```

Comment:
what does Perform mean?

Reply:
This code copy/pasted from first version.

Reply:
Ekhm, but it's not used. Could you delete the class?